packages/manager/src/features/Kubernetes/KubernetesClusterDetail/NodePoolsDisplay/NodePool.tsx

```diff
@@ -65,7 +73,14 @@ export const NodePool = (props: Props) => {
           py: 0,
         }}
       >
-        <Typography variant="h2">{typeLabel}</Typography>
+        <Box display="flex">
+          <Typography variant="h2">{typeLabel}</Typography>
+          <Divider
+            orientation="vertical"
+            sx={(theme) => ({ height: 16, margin: `4px ${theme.spacing(1)}` })}
+          />
+          <Typography variant="h2">{count} Nodes</Typography>
```

Comment:
Let's use `pluralize` here to cover the case when we've got one node.

```suggestion
          <Typography variant="h2">{pluralize('Node', 'Nodes', count)}</Typography>
```
![Uploading Screenshot 2025-01-15 at 1.31.27 PM.jpg…]()

Reply:
Could we also add test coverage to confirm the number of nodes displays when the cluster is created? A good place might be at the end of the tests for a standard cluster and an enterprise cluster in `lke-create.spec.ts`, where we redirect to the cluster details page and confirm things.